BangazonWebApp/BangazonWebApp/Models/Employees.cs

```diff
@@ -0,0 +1,41 @@
+﻿//Author: Lauren Richert
+//Purpose: To reference the Employees table and the values
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonWebApp.Models
+{
+    public class Employees
+    {
+        public int Id { get; set; }
+
+        [Required]
+        [Display(Name = "First Name")]
+        public string FirstName { get; set; }
+
+        [Required]
+        [Display(Name = "Last Name")]
+        public string LastName { get; set; }
+
```

Comment:
According to your ERD, you have a boolean property of `Supervisor` on the employee table too